### ISSUE TITLE
Keep handling OTEL_EXPORTER_OTLP_SPAN/METRIC_ENDPOINT

### ISSUE THREAD

**Issue by carlosalberto** [lightstep/otel-launcher-java#94]
Now that 0.17.0 renamed the final environment variables, we need to handle the old ones as deprecated:

1. Check  `OTEL_EXPORTER_OTL_SPAN_ENDPOINT` and if there's any value set, pass it to `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`. In case they are both set, give preference to the `TRACES` one*
2. Do 1) for Metrics (less important as we are hiding that for now).

*Maybe we can do some logging for that, but that can be done in a future PR (if/as needed).

Edit: This SHOULD be implemented following along the next rules:

a. Get the value of `OTEL_EXPORTER_SPAN_ENDPOINT` (after already appending the `UNSAFE` legacy part, if any).
b. If `OTEL_EXPORTER_OTLP_ENDPOINT` is **not set** by the user, try to use the value from a), if any.

In other words, `OTEL_EXPORTER_OTLP_ENDPOINT` has higher precedence.

**Comment by malafeev** [comment]
we cannot pass any value to `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` because it's not supported by java to set env variables.
`otel.exporter.otlp.endpoint` system property is used to pass endpoint. It's common for trace and metrics exporter: 
1. https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java#L59
2. https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java#L73

**Comment by carlosalberto** [comment]
As discussed today, let's support `_SPAN_ENDPOINT` only for now.

**Comment by malafeev** [comment]
handling is implemented
